### ISSUE TITLE
Add contributers section

### ISSUE THREAD

**Issue by steampixel** [steampixel/SteamPixelSepa#3]
Hey @akshaynikhare feel free to add a contributers section with a link to your profile to the readme.

**Comment by akshaynikhare** [comment]
@steampixel  i have opened another pull request ,

Question , do you know how do i mark an issue with a pull request

**Comment by steampixel** [comment]
no. unfortunately I don't know. I would just copy the link in...
